service/src/main/kotlin/no/nav/su/se/bakover/service/søknad/SøknadServiceImpl.kt

```diff
@@ -70,7 +70,7 @@ internal class SøknadServiceImpl(
                 val søknad = Søknad.Ny(
                     sakId = it.id,
                     id = UUID.randomUUID(),
-                    opprettet = it.opprettet,
+                    opprettet = Tidspunkt.now(),
```

Comment:
Blir dette ekvivalent?
Edit: Ser at `it` her vel er den eksisterende saken som hentes ut, og da blir det jo litt rart å bruke dens opprettet-tidspunkt ja.

Reply:
Det er en bug jeg introduserte som Ramzi spotta :) OG testa!